src/Bravo3/ImageManager/Services/ImageManager.php

```diff
@@ -188,18 +188,20 @@ public function pull(Image $image)
     /**
      * Get meta information about the source image from the cache layer.
      *
-     * @param Image $image
+     * @param Image|string $image
      *
      * @return ImageMetadata
      */
-    public function getMetadata(Image $image)
+    public function getMetadata($image)
     {
         // If the $image is a variation, refer to its parent
         // for the metadata.
         if ($image instanceof ImageVariation) {
             $img_key = $image->getKey(true);
```

Comment:
This introduces a BC break, if an `ImageVariation` was passed, formally it would retrieve metadata of the Image object (which is very intentional) - to support ImageVariation metadata, another function should be created (SOLID-friendly) or an added function argument that allows `->getKey(true)` to be used (kind of a SOLID violation)